rules/autoconsent/transip-nl.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "transip-nl",
+  "runContext": {
+      "urlPattern": "^https://www\\.transip\\.nl/"
+    },
+  "cosmetic": true,
```

Comment:
Cosmetic rules are rules that do not reject cookies, but only hide the banners. Since rules in this PR are actually clicking reject buttons, none of them are cosmetic

Reply:
Ah, I see. Thanks for the explanation. I thought all rules using cosmetic features should be flagged as cosmetic.